.github/workflows/chart_test.yml

```diff
@@ -0,0 +1,87 @@
+name: Chart deployment test
+
+on:
+  pull_request:
+    paths:
+      - "charts/**"
+
+env:
+  DOCKER_BUILDKIT: 1
+
+jobs:
+  deploy:
+    name: deployment test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Helm
+        uses: azure/setup-helm@v3.5
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Initialise k3d
+        run: bash .github/integration/scripts/charts/k3d.sh
+        shell: bash
+
+      - name: build postgres
```

Comment:
Looks OK, but maybe could be sped up by using cache? (Untested, but something like `--cache-from=type=gha --cache-to=type=gha`?)

Best merging and seeing if these builds take long enough that it's worth looking into.

Reply:
Fixed by moving the chart tests into the `build PR container` action